.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  # Only run push on main
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+  # Always run on PRs
+  pull_request:
+
+concurrency:
+  group: '${{ github.head_ref }}-${{ github.workflow }}'
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: ${{ matrix.ci_lint_version }} / JDK ${{ matrix.ci_java_version }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Gradle Wrapper Validation
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Generate cache key
+        run: ./.github/checksum.sh checksum.txt
+
+      - uses: actions/cache@v3
```

Comment:
This isn't a blocker. Have you looked at `gradle/gradle-build-action` ([here](https://github.com/gradle/gradle-build-action)). It automates all of the caching so you don't have to do all of this setup manually.

Reply:
Ah just saw that you do below. Why both?

Reply:
leftover from before that action was ready I think, I can clean this all up a bit

Reply:
[0bb3909](https://github.com/slackhq/compose-lints/pull/4/commits/0bb39092f6b3c7a70b1db18d27e62d1ddb3951fb)